examples/update_users_from_csv.py

```diff
@@ -0,0 +1,107 @@
+'''

```

Comment:
Since this code is almost identical to the usergroups, I have the exact same comments as for the usergroups.

Reply:
Applied changes to both files.